### ISSUE TITLE
The Open All button should display a message on hover

### ISSUE THREAD

**Issue by Jiehong** [darekkay/static-marks#29]
Hi,

The "open all" button for each category has an aria-label, which is great, but is missing a title:

```
<button type="button" class="btn-unstyled" aria-label="Open all links from this bucket" tabindex="-1">
  <div class="icon icon-open-all"></div>
</button>
```

Having a title would allow a on-hover element to be shown for sighter users:

```
<button type="button" class="btn-unstyled" aria-label="Open all links from this bucket" tabindex="-1" title="Open all links from this bucket">
  <div class="icon icon-open-all"></div>
</button>
```

**Comment by RishiKumarRay** [comment]
Can I work on this changes, assign me

**Comment by RishiKumarRay** [comment]
Can you guide me, where the files are present so that i can make the changes

**Comment by darekkay** [comment]
@RishiKumarRay Sure, here it is: https://github.com/darekkay/static-marks-app/blob/master/src/components/bucket/Bucket.jsx#L18

**Comment by RishiKumarRay** [comment]
If you approve i will raise PR

**Comment by darekkay** [comment]
Yes, looks good to me

**Comment by RishiKumarRay** [comment]
fixed and raise PR

**Comment by darekkay** [comment]
PR merged: https://github.com/darekkay/static-marks-app/pull/3

@all-contributors please add @RishiKumarRay to maintenance

@all-contributors please add @Jiehong to maintenance

**Comment by darekkay** [comment]
@all-contributors please add @RishiKumarRay for maintenance

@all-contributors please add @Jiehong for maintenance

**Comment by darekkay** [comment]
@all-contributors please add @RishiKumarRay for maintenance

**Comment by darekkay** [comment]
@all-contributors please add @Jiehong for maintenance

**Comment by darekkay** [comment]
Released in [2.4.0](https://github.com/darekkay/static-marks/releases/tag/v2.4.0)
